languages/swift/config.json

```diff
@@ -55,10 +55,10 @@
         "prerequisites": ["numbers", "conditionals"]
       },
       {
-        "slug": "strings",
+        "slug": "string-components",
         "uuid": "082d7aeb-ee25-4217-bd6f-82532a65797a",
-        "concepts": ["strings"],
-        "prerequisites": ["arrays"]
+        "concepts": ["opaque-indices", "string-components", "string-indexing"],
```

Comment:
My last suggestion is to add the importing concept here too.

Reply:
Importing concept added.

Reply:
Nice

Reply:
Are you able to take care of the dead link issue described above?

Reply:
@wneumann For renames, I'd just edit that file (it will be overwritten after merging).